### PR TITLE
Upgrade dotnet core version to 6

### PR DESCRIPTION
**Pull request by rafael-santiago-take**
Upgrade dotnet core version to 6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The **BLiP C# SDK** is a set of Nuget packages that allow the creation of [BLiP]
 
 ## Requirements
 
-- .NET Core 3.0 or above (download the SDK [here](https://dotnet.microsoft.com/download))
+- .NET Core 3.1 or above (download the SDK [here](https://dotnet.microsoft.com/download))
```
Comment (caiopsouza) [inline README.md:8]: Should it be 6?

File: src/Samples/Builder.Console/Builder.Console.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```
Comment (caiopsouza) [inline src/Samples/Builder.Console/Builder.Console.csproj:5]: Why was `netcoreapp3.1` kept?

Comment (rafael-santiago-take) [reply src/Samples/Builder.Console/Builder.Console.csproj:5]: that way, the project is compatible to be used in dotnet core 3.1 projects too

File: README.md (hunk 1 above)
Comment (rafael-santiago-take) [reply README.md:8]: no... 3.1 or "above"

**Review by caiopsouza — APPROVED** [review]

**Pull request merged**